listener.go

```diff
@@ -35,7 +34,7 @@ func newListener(addr ma.Multiaddr, transport tpt.Transport, localPeer peer.ID,
 	if err != nil {
 		return nil, err
 	}
-	conn, err := net.ListenUDP(lnet, laddr)
+	conn, err := t.connManagers.Listen(lnet, laddr)
```

Comment:
Please don't access unexported fields of other structs.

Reply:
I should have been more specific. Please don't access **any** fields of other structs.